tcpdf.php

```diff
@@ -2366,8 +2366,8 @@ public function setSpacesRE($re='/[^\S\xa0]/') {
 
 	/**
 	 * Enable or disable Right-To-Left language mode
-	 * @param Boolean $enable if true enable Right-To-Left language mode.
-	 * @param Boolean $resetx if true reset the X position on direction change.
+	 * @param boolean $enable if true enable Right-To-Left language mode.
```

Comment:
```suggestion
	 * @param bool $enable if true enable Right-To-Left language mode.
```

Reply:
Let's let @nicolaasuni decide :)
After that I will make the code base use long or short hints

Reply:
Follow PSR-5, basically use the name you would use for types in code. The same for `int` (vs integer), `float` (vs. double) etc.